### PR TITLE
Initial commit: Hello world

### PR DESCRIPTION
**Pull request by mikesavior**
Summary: Add a Hello World to bff to acclimate one's self with its basic workings.

Test Plan: TBD

### CODE REVIEW

File: infra/bff/friends/hello_world.bff.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "echo",
+    "Hello " + (name === undefined ? "world" : name) + "!"
+  ]);
+}
+
+register("helloWorld", "Hello world to acclimate with bff sys", () => {
+  return helloWorld();
+});
+
+register(
```
Comment (randallb) [inline infra/bff/friends/hello_world.bff.ts:17]: this seems like a bug?

**Review by randallb — CHANGES_REQUESTED** [review]

File: infra/bff/friends/hello_world.bff.ts (hunk 1 above)
Comment (mikesavior) [reply infra/bff/friends/hello_world.bff.ts:17]: Aha, this commented line? If so, yes, whoops. lemme nix that

Comment (randallb) [reply infra/bff/friends/hello_world.bff.ts:17]: I don't get why there are two register calls.

Comment (mikesavior) [reply infra/bff/friends/hello_world.bff.ts:17]: OH. Yeah! Yeah that's a me-thing. Gotcha.

Comment (mikesavior) [reply infra/bff/friends/hello_world.bff.ts]: Updated to remove that extra register() call

**Review by randallb — APPROVED** [review]
yay you did it

**Pull request merged**